packages/palette/src/elements/Select/Select.tsx

```diff
@@ -220,9 +192,34 @@ const Container = styled(Box)<ContainerProps>`
           cursor: default;
           ${SELECT_STATES.disabled}
         }
+
+        &:not(:has(option[value=""]:checked)):not(:focus) {
+          ${!(props.disabled || props.focus) && SELECT_STATES.completed}
+          ${props.error && SELECT_STATES.error}
+        }
+
+        &:has(option[value=""]:checked) {
+          ${props.title &&
+          css`
+            color: transparent;
+          `}
+        }
       `
     }};
   }
 
   ${caretMixin}
 `
+
+const StyledLabel = styled.label`
+  position: absolute;
+  top: 50%;
+  left: 5px;
+  padding: 0 5px;
+  pointer-events: none;
+  transform: translateY(-50%);
+  transition: 0.25s cubic-bezier(0.64, 0.05, 0.36, 1);
```

Comment:
The animation feels a bit unnatural on this - maybe try this setting? 

<img width="883" alt="Screenshot 2023-05-11 at 8 57 00 AM" src="https://github.com/artsy/palette/assets/236943/5a1fbfdf-8a00-4d31-a5a7-da203c5330ee">

```
transition: all 500ms cubic-bezier(0.230, 1.000, 0.320, 1.000); /* easeOutQuint */
```

This is a good tool to play with animations: https://matthewlein.com/tools/ceaser

@dzucconi keeps bringing it up, but we really need some standard guidelines on animation!